pyomo/core/base/label.py

```diff
@@ -9,13 +9,15 @@
 #  ___________________________________________________________________________
 
 __all__ = ['CounterLabeler', 'NumericLabeler', 'CNameLabeler', 'TextLabeler',
-           'AlphaNumTextLabeler','NameLabeler', 'CuidLabeler']
+           'AlphaNumericTextLabeler','NameLabeler', 'CuidLabeler']
 
 import six
 if six.PY3:
-    _string = str
+    _translate = str.translate
 else:
-    import string as _string
+    import string
+    _translate = string.translate
```

Comment:
Just being more specific about the thing we were really trying to import: what we really want is the `translate()` method.  I was actually surprised this wasn't handled by `six.moves`